types/should/index.d.ts

```diff
@@ -88,10 +88,12 @@ interface ShouldAssertion {
   keys(...allKeys: string[]): ShouldAssertion;
   keys(allKeys: string[]): ShouldAssertion;
   enumerable(property: string, value?: any): ShouldAssertion;
+  size(size: number): ShouldAssertion;
+  value(key: string | number, value: any): ShouldAssertion;
   enumerables(...properties: string[]): ShouldAssertion;
   startWith(expected: string, message?: any): ShouldAssertion;
   endWith(expected: string, message?: any): ShouldAssertion;
-  throw(message?: any): ShouldAssertion;
+  throw(message?: any, properties?: Object): ShouldAssertion;
```

Comment:
I think instead of `Object` you want `{}` or `object`. `object` doesn't permit things like `number` or `string`